src/storybook/stories/Section.stories.tsx

```diff
@@ -191,12 +208,25 @@ export const RadioSection: Meta<SectionMetaProps> = {
         args={{
           ...args,
           children: (
-            <RadioGroupSection
-              items={['Radio group option 1', 'Radio group option 2']}
-              keyExtractor={(s) => s}
-              itemToText={(t) => t}
-              selected="Radio group option 1"
-            />
+            <View style={{rowGap: 20}}>
+              <RadioGroupSection
+                items={['Radio group option 1', 'Radio group option 2']}
+                keyExtractor={(s) => s}
+                itemToText={(t) => t}
+                selected="Radio group option 1"
+              />
+
+              <RadioGroupSection
+                items={[
+                  'Spacious radio group option 1',
+                  'Spacious radio group option 2',
+                ]}
+                keyExtractor={(s) => s}
+                itemToText={(t) => t}
+                selected="Spacious radio group option 1"
+                type="spacious"
```

Comment:
It's important to return `RadioGroupSection` as the root node here, not the `View`, since `args` are applied to it. `args` is where the user selection from storybook comes from, including the `type` param. 

That is, I think we can keep this like it was, and have `type` as an storybook option, not a hard coded variant.